### DISCUSSION TITLE
Statistics for listings

### DISCUSSION THREAD

**Discussion by andypiper** [forem/forem#16309] [Feature Requests]
As a creator of listing entries, it would be helpful to be able to see more information about engagement and display for the listing entries. I think this could be seen as a reasonable value exchange particularly in the case where the listings either use earned points, or paid-for points. I can already see metrics around my posts in my dashboard, but (unless I am missing something) the ability to manage listings entries is limited to edit (for 24 hours), bump (costing points), or unpublish.

This could be potentially worked on alongside something like [the ability to set working expiration dates](https://github.com/forem/forem/discussions/15330) / if there was a wider set of improvements to listings.

**Comment by JennieOcken** [comment]
We are currently thinking through the dashboard holistically to improve some of those things. I'm not sure yet where we are going to end up but I am going to cc: @ludwiczakpawel to be aware of this discussion and put this in the "up next" area as we are committed to working on dashboard soon.
